clang/lib/CIR/CodeGen/CIRGenExprConst.cpp

```diff
@@ -1063,22 +1063,27 @@ class ConstantLValueEmitter
       Indices.push_back(Attr);
     }
 
+    if (Indices.empty())
+      return {};
```

Comment:
This one is just to avoid printing empty indices in CIR output (`#cir.global_view<@x, []>`), right?

Reply:
That's right!